rotkehlchen/globaldb/handler.py

```diff
@@ -2103,3 +2109,12 @@ def delete_location_asset_mappings(entries: list[LocationAssetMappingDeleteEntry
                         f'Failed to delete the location asset mapping of {entry.location_symbol} in '  # noqa: E501
                         f'{entry.location} because it does not exist in the DB.',
                     )
+
+    @staticmethod
+    def get_unsupported_assets(location: 'Location') -> set[str]:
```

Comment:
So this is not a good approach. Wherever you use this you need to change this.

What you are currently doing is readign all asset unsupported symbols into memory. And at every place you call it then you do an inclusion comparison in python.

This is bad.

What you need to turn this function to is `is_asset_symbol_unsupported(location: 'Location') -> bool`

And in here turn the sql query to an inclusion query. Faster and nothing loaded in memory.

Isn't this how you did the location_mappings too? Why follow this approach here?